src/validations/Validation.java

```diff
@@ -0,0 +1,20 @@
+package validations;
+
+public class Validation {
+
+    public static void notNull(String field, String error) {
+        if (field == null)
+            throw new IllegalArgumentException(error);
+    }
+
+    public static void notBlank(String field, String error) {
+        if (field.isEmpty() || field.isBlank())
+            throw new IllegalArgumentException(error);
+    }
+
+    public static void isValidNumberHours(int number, String error) {
+        if (number <= 1 || number > 20)
+            throw new IllegalArgumentException(error);
+    }
```

Comment:
Não esquece que, segundo a especificação da tarefa da implementação do Course, a validação deve passar se o número de horas estimadas `x` estiver entre 1 e 20 (intervalo inclusivo). Do jeito que está agora, se a duração estimada for de 1h, não vai passar na validação.